tdrs-backend/tdpservice/settings/common.py

```diff
@@ -475,6 +475,13 @@ class Common(Configuration):
                 'expires': 15.0,
             },
         },
+        'Deactivate Users': {
+            'task': 'tdpservice.email.tasks.deactivate_users',
```

Comment:
@raftmsohani what's the name of this periodic task? Is it `Deactivate Users`? I'm not seeing it in the list below, and am unsure how to trigger the task after changing my test user's last login date. I was assuming after setting the last login date to some date > 180 days ago, I would change one of these periodic tasks to trigger the deactivation and email notification...? 

![periodic_tasks](https://github.com/user-attachments/assets/19c42d3b-a30d-4d7f-9748-0b3536061c1d)

Reply:
@raftmsohani Im gonna deploy this. just realized i was testing this on another branch 😆